test/test_service.py

```diff
@@ -0,0 +1,25 @@
+from typing import List
+
+from packaging.version import Version
+
+import pip_audit.service as service
+
+
+def test_osv():
+    osv = service.OsvService()
```

Comment:
What do you think about invoking the actual API as part of the test? Do you think it's ok.

I'll write a proper fixture and get those last few lines of coverage to get the build passing but I just wanted to run this by you first.

Reply:
I think it's okay!

We can put the networked tests behind a `pytest.mark` + CLI option if we're concerned about the OSV API's reliability: https://stackoverflow.com/questions/47559524/pytest-how-to-skip-tests-unless-you-declare-an-option-flag

Reply:
Sounds good. I've intentionally avoided being too specific about what we get from the API to avoid failures if vulnerabilities get added to a package or modified, etc.